### PR TITLE
register step 1/2

### PR DESCRIPTION
**Pull request by Levsha1337**

### CODE REVIEW

File: MarathonWPF/App.config [hunk 1]
```diff
@@ -3,8 +3,9 @@
     <configSections>
     </configSections>
     <connectionStrings>
+            <!--connectionString="Data Source=(localdb)\MSSQLLocalDB;Initial Catalog=g463_runners;Integrated Security=True"-->
         <add name="MarathonWPF.Properties.Settings.g463_runnersConnectionString"
-            connectionString="Data Source=(localdb)\MSSQLLocalDB;Initial Catalog=g463_runners;Integrated Security=True"
+             connectionString="Data Source=shuraga.ru;Initial Catalog=g463_runners;Persist Security Info=True;User ID=superuser;Password=P@$$w0rd"
```
Comment (Levsha1337) [inline MarathonWPF/App.config:8]: Remote server connection

File: MarathonWPF/ProfileControl/Register.xaml.cs [hunk 2]
```diff
@@ -1,4 +1,6 @@
-﻿using System.IO;
+﻿using Microsoft.Win32;
+using System;
```
Comment (Levsha1337) [inline MarathonWPF/ProfileControl/Register.xaml.cs:2]: DO NOT REMOVE UNUSED NAMESPACES

File: MarathonWPF/RegisterOnMarathon.xaml [hunk 3]
```diff
@@ -33,14 +33,14 @@
...
+            <RadioButton Content="Вариант B" HorizontalAlignment="Left" Margin="459,204,0,0" VerticalAlignment="Top" Foreground="Gray" FontSize="22" GroupName="Variants" />
+            <RadioButton Content="Вариант C" HorizontalAlignment="Left" Margin="459,240,0,0" VerticalAlignment="Top" Foreground="Gray" FontSize="22" GroupName="Variants" />
             <Label Content="$185" HorizontalAlignment="Left" Margin="518,347,0,0" VerticalAlignment="Top" Foreground="Gray" FontSize="40"/>
             <Label Content="Взнос:" HorizontalAlignment="Left" Margin="147,316,0,0" VerticalAlignment="Top" Foreground="Gray" FontSize="18"/>
             <Label Content="Сумма взноса:" HorizontalAlignment="Left" Margin="79,361,0,0" VerticalAlignment="Top" Foreground="Gray" FontSize="18"/>
-            <TextBox HorizontalAlignment="Left" Height="23" Margin="221,371,0,0" TextWrapping="Wrap" Text="TextBox" VerticalAlignment="Top" Width="120"/>
-            <ComboBox HorizontalAlignment="Left" Margin="221,324,0,0" VerticalAlignment="Top" Width="120" RenderTransformOrigin="0.508,0.122"/>
```
Comment (Levsha1337) [inline MarathonWPF/RegisterOnMarathon.xaml:43]: Stop using XAML constructor!!!

**Review by Levsha1337 — COMMENTED** [review]
Its okay to merge

File: MarathonWPF/RegisterOnMarathon.xaml (hunk 3 above)
Comment (Nismoteika) [reply MarathonWPF/RegisterOnMarathon.xaml:43]: it's touchpad fail, when i want move he create rendertransform

File: MarathonWPF/ProfileControl/Register.xaml.cs (hunk 2 above)
Comment (Nismoteika) [reply MarathonWPF/ProfileControl/Register.xaml.cs:2]: it's recomended by vs:D

**Pull request merged**